activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1394,6 +1394,12 @@ def create_schema_dumper(options) # :nodoc:
       end
 
       private
+        def validate_change_column_null_argument!(value)
+          unless value == true || value == false
+            raise "change_column_null expects a boolean value (true for NULL, false for NOT NULL). Got: #{value}"
```

Comment:
Should we raise an `ArgumentError`?
`Got: #{value}` => `Got: #{value.inspect}`

Reply:
Yeah good call. However it still ends up as a `StandardError` by the time it gets to the migration. https://github.com/rails/rails/blob/920fcce54c9e112dc724d587e58182c5603de2d0/activerecord/lib/active_record/migration.rb#L1372